web/src/routes/(user)/explore/+page.server.ts

```diff
@@ -9,11 +9,13 @@ export const load = (async ({ locals, parent }) => {
   }
 
   const { data: items } = await locals.api.searchApi.getExploreData();
-  const { data: people } = await locals.api.personApi.getAllPeople();
+  const { data: people } = await locals.api.personApi.getAllPeople({ areHidden: true });
+  const { data: countpeople } = await locals.api.personApi.getPersonCount();
```

Comment:
You already have all the people in the above line, so can we get the stats from that no?

Reply:
Same thing, I need to know the number of hidden / unhidden faces. Indeed with `getAllPeople({ areHidden: true })` the client can determine how many hidden / unhidden faces there is, but I prefered to get that information directly from the server to avoid taking too many resources. Should I move the logic to the client ?

Reply:
If you already have all the people in memory, I don't think it is hard to calculate or resource intensive.

Alternatively we could change the get all endpoint to return { people: Dto[], total: number, hidden: number } etc.

Reply:
What should be the name for this dto ? PersonAllResponseDto ?

Reply:
Maybe People Response Dto with a list of person response dto.